spec/index.bs

```diff
@@ -10856,52 +10938,55 @@ dictionary GPURenderPassColorAttachment {
-        - |resolveTexture|.{{GPUTextureDescriptor/format}} must equal
-            |renderTexture|.{{GPUTextureDescriptor/format}}.
-        - |resolveViewDescriptor|.{{GPUTextureViewDescriptor/format}} must support resolve according to [[#plain-color-formats]].
+                - |this|.{{GPURenderPassColorAttachment/depthSlice}} must not [=map/exist|be provided=].
+
+            - If |this|.{{GPURenderPassColorAttachment/loadOp}} is {{GPULoadOp/"clear"}}:
+                - Converting the IDL value |this|.{{GPURenderPassColorAttachment/clearValue}}
+                    [$to a texel value of texture format$] |renderViewDescriptor|.{{GPUTextureViewDescriptor/format}}
+                    must not throw a {{TypeError}}.
+
+                    Note: An error is not thrown if the value is out-of-range for the format but in-range for
+                    the corresponding WGSL primitive type (`f32`, `i32`, or `u32`).
+            - If |this|.{{GPURenderPassColorAttachment/resolveTarget}} is [=map/exist|provided=]:
+                - |renderTexture|.{{GPUTexture/sampleCount}} must be &gt; 1.
+                - |resolveTexture|.{{GPUTexture/sampleCount}} must be 1.
+                - |this|.{{GPURenderPassColorAttachment/resolveTarget}} must be a [$renderable texture view$].
+                - |this|.{{GPURenderPassColorAttachment/resolveTarget}}.{{GPUTextureView/[[renderExtent]]}} and
+                    |this|.{{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[renderExtent]]}} must match.
+                - |resolveViewDescriptor|.{{GPUTextureViewDescriptor/format}} must equal
+                    |renderViewDescriptor|.{{GPUTextureViewDescriptor/format}}.
+                - |resolveTexture|.{{GPUTextureDescriptor/format}} must equal
+                    |renderTexture|.{{GPUTextureDescriptor/format}}.
+                - |resolveViewDescriptor|.{{GPUTextureViewDescriptor/format}} must support resolve according to [[#plain-color-formats]].
+        </div>
 </div>
 
-<div algorithm>
+<div algorithm data-timeline=device>
     A {{GPUTextureView}} |view| is a <dfn abstract-op>renderable texture view</dfn>
     if the following requirements are met:
 
-    - |view|.{{GPUTextureView/[[texture]]}}.{{GPUTexture/usage}}
-        must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
-    - |descriptor|.{{GPUTextureViewDescriptor/dimension}} must be either {{GPUTextureViewDimension/"2d"}} or {{GPUTextureViewDimension/"3d"}}.
-    - |descriptor|.{{GPUTextureViewDescriptor/mipLevelCount}} must be 1.
-    - |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} must be 1.
-    - |descriptor|.{{GPUTextureViewDescriptor/aspect}} must refer to all [=aspects=] of
-        |view|.{{GPUTextureView/[[texture]]}}.
-
-    where |descriptor| is |view|.{{GPUTextureView/[[descriptor]]}}.
+    1. Let |descriptor| be |view|.{{GPUTextureView/[[descriptor]]}}.
+    1. |view| is not renderable if any of the following conditions are unsatisfied:
```

Comment:
This reads a bit funny because it says "following requirements" then launches into some steps and then restates what the requirements are for.

How about #4570-style:
```
... if the requirements in the following steps are met:

1. Let |descriptor| be |view|.{{GPUTextureView/[[descriptor]]}}.
1. |view|.{{GPUTextureView/[[texture]]}}.{{GPUTexture/usage}}
    must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
...
```

The `validusage` block could either be around all the steps, or on the whole algorithm.

Reply:
Good catch! I thought about that when I wrote it, told myself I was going to come back and refactor and then 👋  *gestures vaguely at all of the everything else*.

Refactored to follow the #4570 style more.